config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -197,6 +197,7 @@ periodics:
     testgrid-tab-name: bazel-build-master
     description: run bazel build //... -//vendor/...
   interval: 5m
+  cluster: k8s-infra-prow-build
```

Comment:
@ameukam just a heads up on the test failure, all `k8s-infra-prow-build` jobs require resource limits and requests as part of https://github.com/kubernetes/test-infra/issues/18530

Reply:
Thanks for the heads up. I'll update the PR when we don't need to hold it.

Reply:
Added in #18629 #18630.